config.js

```diff
@@ -4,7 +4,7 @@ let config = {
     //info variables
     version: process.env.VERSION,
     date: process.env.DATE,
-
+    inactive_user_days : process.env.INACTIVE_APPLICATION_DAYS ? process.env.INACTIVE_APPLICATION_DAYS : 45,
```

Comment:
This pattern of assigning a value to a variable if it is null can be replaced with:
`inactive_user_days : process.env.INACTIVE_APPLICATION_DAYS || 45`

Reply:
Updated.